sentry-android-core/src/main/java/io/sentry/android/core/AppStartState.java

```diff
@@ -13,6 +13,9 @@ public final class AppStartState {
 
   private static @NotNull AppStartState instance = new AppStartState();
 
+  /** // We filter out App starts more than 60s */
+  private static final int MAX_APP_START = 60000;
```

Comment:
I'm a big fan of adding units to constant names so you don't have to go looking through usages.

```suggestion
  private static final int MAX_APP_START_MILLIS = 60000;
```